### ISSUE TITLE
New authentication mechanism

### ISSUE THREAD

**Issue by mdisabato** [chefhl/homebridge-plugin-netatmo-rain-sensor#4]
**Is your feature request related to a problem? Please describe:**
Netatmo changed the authentication mechanism used to OAuth in October.

https://dev.netatmo.com/apidocumentation/oauth#authorization-code

**Describe the solution you'd like:**
If possible, this plugin should be updated to conform to the new method. This has been done in https://github.com/skrollme/homebridge-eveatmo and might be able to be ported here.

**Comment by chefhl** [comment]
I totally agree with this. Sad thing is I'm terribly busy with my daily job so I cannot give an estimate on when I will be able to do it. I really wish I would have more time for this. I am definitely open for any open source community contribution in this matter.

**Comment by GM7722** [comment]
Happy to make a small Donation for making this plugin work with the new authentication :-)

**Comment by Jeanke** [comment]
I also realy would like it to work again, it whas so usefull to me. But I realy understand the thing of not having enough hours in one day.
I would help making the changes myselve, but this is above my knowledge. ;-)
Would begging help to make the changes? :-)
